portfolio/src/main/webapp/script.js

```diff
@@ -32,6 +32,29 @@ function addRandomQuote() {
   const quote = quotes[Math.floor(Math.random() * quotes.length)];
 
   // Add it to the page.
-  const spongebobContainer = document.getElementById('spongebob-container');
-  spongebobContainer.innerText = quote;
+  const spongebobContainer = document.getElementById('quote-container');
+
+  let bubblesImg = document.getElementById('bubbles');
+  let i = 1;
+  function bubbleLoop() {
```

Comment:
what does this do? what significance do 20 and 41 have?

Reply:
41 is for the loop to end so that it runs 40 times. 20 is the halfway point of the iterations at which I show the text so that the animation doesn't occur after the text shows up. I randomly chose it as such so that it doesn't run for too long while also keeping the animation relatively smooth.

Reply:
bubbleLoop() in general makes an image of bubbles go from the bottom of the div through to the top so that it looks like bubbles are floating upwards.

Reply:
can you add a comment explaining what bubbleLoop() does? since it's not immediately obvious.

also, it's okay that 40 and 20 are arbitrary, but define a constant (numLoops or similar) and use that instead. (numLoops +1, numLoops/2, etc). otherwise it's confusing to the reader what these numbers mean.

Reply:
this looks better, thanks.